### ISSUE TITLE
Loops excercise solution 

### ISSUE THREAD

**Issue by Noe-Santana** [ronreiter/interactive-tutorials#585]
I´m working on the spanish version, maybe the variable "numeros" is diferent in english, but the code should be the same. 
On the expected response is not included the 412 but on the description of the excersice says that only numbers greater than 412 should be excluded. 

 y=0
for x in numeros: 
    if numeros[y] % 2 ==0 and numeros[y] <= 412:
        print (numeros[y])
    y += 1

**Comment by s0rta** [comment]
yeah the solutions seem to be based on a code comparison not an output comparison, something that would require a bit of a site overhaul it seems.
